jenkins/param-parallel-mtr.yml

```diff
@@ -1,5 +1,5 @@
 - job:
-    name: percona-server-8.0-param-parallel-mtr
+    name: percona-server-8.0-param-parallel-mtr-ps8556
```

Comment:
I think we don't want to change job name. In all places it should stay `percona-server-8.0-param-parallel`.